QUEST.tsv

```diff
@@ -2765,252 +2765,252 @@ QUEST_20151016_002772	$When the Royal Army said that they will spare our lives w
+QUEST_20151102_002933	I saw Orsha from a slope on the way here from a town.{nl}The splendor surrounded by castle wall I've seen as a kid has disappeared and now coiled with trees with the size of a house.{nl}I wonder how long you can last in such a place.
+QUEST_20151102_002934	It's obviously intelligence for wizard and its affiliation.{nl}However, you also need to remember spirit to cast spells frequently.{nl}Select well between balance and concentration.
+QUEST_20151102_002935	It's obviously strength for archer and its affiliation.{nl}However, you cannot forget nimble dexterity.{nl}It's best to think ahead of what to become.
+QUEST_20151102_002936	It's obviously intelliegence for cleric and its affiliation.{nl}However, you may also require strength or spirit.{nl}It's best to think ahead of what to become.
+QUEST_20151102_002937	Only if our town also had an inn for merchants to stay and rest.{nl}We could've prepared beforehand..{nl}After spending so long at an inlet of a valley, we got to suffer for not knowing the world affairs.
+QUEST_20151102_002938	Living there inside of Orsha's partially destroyed fortress.{nl}Living here with swarming monsters and fear of safety.{nl}Both are tough choices to make.{nl}I wish to return back to my hometown..
+QUEST_20151102_002939	Please let the people with the smell of the ocean to head down to Dallanas.{nl}It's starting to become uneasy with the people not to be seen as time passes.{nl}By any chance, did they all leave to Orsha without me?
+QUEST_20151102_002940	There's no way that such a huge amount of people can enter Orsha.{nl}That's why we try to gather and accommodate them here as much as possible.
+QUEST_20151102_002941	Lethargic and fear can be seen in people's eyes.{nl)What would you do if you were the village chief?
+QUEST_20151102_002942	{nl}It's the elderly's complaint..{nl}I worry for the lord..
+QUEST_20151102_002943	Those who are usually lazy are still carefree in these time of danger.{nl}People who were suppose to arrive earlier has yet to come.
+QUEST_20151102_002944	The land has been protected by the ancestors of the family.{nl}I will protect the land, no matter the trouble.{nl}Please lend me your power.
+QUEST_20151102_002945	There has been no case of demons being involved during the several years I spent as a chaser.{nl}I believe we have seen all the demons within the area.
+QUEST_20151102_002946	The rumors about you are incredible.{nl}It has spread all around Orsha.
+QUEST_20151102_002947	I will have to revisit my hometown after this task.{nl}I wish to see my parents after experiencing such a horrible event.
+QUEST_20151102_002948	I don't believe there are more important task over at Orsha than here.
+QUEST_20151102_002949	I pray that all the informants are safe.
+QUEST_20151102_002950	To live in such a world, chaser should be more suitable than a hunter.{nl}
```

Comment:
isn't it to survive in such a world (this world)?
chaser should be more suitable than a hunter -> a chaser ...